server/src/pages/shot/controller.js

```diff
@@ -116,6 +116,27 @@ exports.deleteShot = function(shot) {
   req.send(`id=${encodeURIComponent(shot.id)}&_csrf=${encodeURIComponent(model.csrfToken)}`);
 };
 
+exports.saveEdit = function(shot, shotUrl) {
+  let url = model.backend + "/api/save-edit";
+  let req = new XMLHttpRequest();
```

Comment:
We should use the fetch API for any new code: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API